docs/rules/self-closing-comp.md

```diff
@@ -17,5 +17,79 @@ var contentContainer = <div className="content"></div>;
 
 var HelloJohn = <Hello name="John" />;
 
-var Profile = <Hello name="John"><img src="picture.png" /></Hello>; 
+var Profile = <Hello name="John"><img src="picture.png" /></Hello>;
+```
+
+## Rule Options
+
+It takes an option as the second parameter, which corresponding what components tags should be self-closed when this is possible.
+
+```js
+...
+"self-closing-comp": [<enabled>, 'all'|'component'|'html'>]
```

Comment:
i wonder if rather than "all", this should have the format `{ component: true, html: true }` - that allows for other categorizations, and works just as well for the default of `{ component: true, html: false }`